conf.js

```diff
@@ -1 +1,9 @@
 //conf.js
+exports.config = {
+
+    directConnect = true,
+    framework: 'jasmine2',
+    seleniumAddress: 'http://localhost:4444/'
+    specs: ['spec.js']
+
+}
```

Comment:
leave new line at end

Reply:
both directConnect and Selenium webserver address not require. Comment directConnect